index.md

```diff
@@ -32,13 +32,20 @@ $ helm repo update
 {{ latest_chart.description }}
 
 ```console
-$ helm install {{ site.repo_name }}/{{ latest_chart.name }} --name myrelease --version {{ latest_chart.version }}
+$ helm install myrelease {{ site.repo_name }}/{{ latest_chart.name }} --version {{ latest_chart.version }}
 ```
 
-| Chart Version | App Version | Date |
-| ------------- | ----------- | ---- |
+You can then use the `HelmCluster` cluster manager in [dask-kubernetes](https://kubernetes.dask.org/en/latest/) to manage scaling form your Python session.
+
+```python
+from dask_kubernetes import HelmCluster
 
+cluster = HelmCluster(release_name="myrelease")
+cluster.scale(10)
+```
```

Comment:
This section is aimed at the dask helm chart rather than daskhub, right? If so, it would benefit from a clarification.

Reply:
Good point thanks! This PR was raised before daskhub was added.